src/test/docker/centos7/Dockerfile

```diff
@@ -28,6 +28,7 @@ RUN yum -y update \
       ccache \
       cppcheck \
       czmq-devel \
+      libuuid-devel \
```

Comment:
uuid-devel, libuuid-devel are already listed below. (Though I do wonder why we currently have both names, seems like one should be invalid.)